common/mariadb/templates/initdb/_init.sql.tpl

```diff
@@ -1,3 +1,25 @@
-CREATE DATABASE IF NOT EXISTS {{.Values.name}} CHARACTER SET utf8 COLLATE utf8_general_ci;
-GRANT ALL PRIVILEGES ON {{.Values.name}}.* TO {{.Values.global.dbUser}}@localhost IDENTIFIED BY '{{include "db_password" .}}';
-GRANT ALL PRIVILEGES ON {{.Values.name}}.* TO {{.Values.global.dbUser}}@'%' IDENTIFIED BY '{{include "db_password" .}}';
+{{- if not .Values.databases }}
+CREATE DATABASE IF NOT EXISTS {{ .Values.name }};
```

Comment:
The `CHARACTER SET utf8 COLLATE utf8_general_ci` should be left in.
Otherwise the default character set of the server will be used.
Checked this for Nova in QA and the server default is `latin1` but the nova db is created with `utf8`:
```
show variables like '%character_set_server%';
+----------------------+--------+
| Variable_name        | Value  |
+----------------------+--------+
| character_set_server | latin1 |
+----------------------+--------+
```
```suggestion
CREATE DATABASE IF NOT EXISTS {{ .Values.name }} CHARACTER SET utf8 COLLATE utf8_general_ci;
```

Reply:
In order to only have a single place where the character-set and collation is set, I added this in the front:
```
SET character_set_server = '{{ .Values.character_set_server }}';
SET collation_server = '{{ .Values.collation_server }}';
``` 

I hope, that's fine with you as well.

Reply:
No, I added it after your proposal.